CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How to Contribute
+
+We welcome pull requests from everyone. We do expect everyone to adhere to the [Mozilla Community Participation Guidelines][participation].
```

Comment:
The declaration of `[participation]` is missing

Reply:
I forgot markdown files even *supported* that.  Will fix and push.